.evergreen/buildvariants-and-tasks.in.yml

```diff
@@ -0,0 +1,460 @@
+    commands:
+      - func: clone
+      - func: test-connectivity
+        vars:
+          debug: 'compass*,electron*,hadron*,mongo*'
+
+  - name: test-csfle
+    tags: ['required-for-publish', 'run-on-pr']
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+      - func: test-csfle
+        vars:
+          debug: 'compass*,electron*,hadron*,mongo*'
+
+  - name: e2e-coverage
+    tags: ['run-on-pr']
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+      - func: e2e-coverage
+        vars:
+          debug: 'compass-e2e-tests*,electron*,hadron*,mongo*'
+
+  - name: generate-vulnerability-report
+    tags: ['required-for-publish', 'run-on-pr']
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+      - func: generate-vulnerability-report
+        vars:
+          debug: 'compass*,electron*,hadron*,mongo*'
+
+  # Publish happens in one go to make sure we are not creating multiple github
+  # releases in parallel
+  - name: publish
+    # Does a dry-run: doesn't actually publishes, but checks that all files
+    # exist and we are not trying to do a "broken" publish
+    tags: ['run-on-pr']
+    depends_on:
+      - name: '.required-for-publish'
+        variant: '*'
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+        vars:
+          scope: mongodb-compass
+      - func: get-all-artifacts
+      - func: publish
+
+  - name: publish-packages-next
+    tags: []
+    depends_on:
+      - name: '.required-for-publish'
+        variant: '*'
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+      - func: publish-packages-next
+
+  - name: publish-dev-release-info
+    tags: []
+    depends_on:
+      - name: 'publish'
+        variant: '*'
+    commands:
+      - func: prepare
+      - func: publish-dev-release-info
+
+  - name: test-web-sandbox-chrome
+    tags: ['required-for-publish', 'run-on-pr']
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+        vars:
+          scope: 'compass-e2e-tests'
+      - func: apply-compass-target-expansion
+        vars:
+          compass_distribution: compass
+      - func: test-web-sandbox
+        vars:
+          mongodb_version: 'latest-alpha-enterprise'
+          browser_name: 'chrome'
+          compass_distribution: compass
+          debug: 'compass-e2e-tests*,electron*,hadron*,mongo*'
+
+  - name: test-web-sandbox-firefox
+    tags: ['required-for-publish', 'run-on-pr']
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+        vars:
+          scope: 'compass-e2e-tests'
+      - func: apply-compass-target-expansion
+        vars:
+          compass_distribution: compass
+      - func: test-web-sandbox
+        vars:
+          mongodb_version: 'latest-alpha-enterprise'
+          browser_name: 'firefox'
```

Comment:
That's the only difference between chrome and firefox tasks. Maybe can also use a `for` here?